contracts/test/utils/TestFlowLimitLiveNetwork.sol

```diff
@@ -53,21 +53,29 @@ contract TestFlowLimitLiveNetwork is IFlowLimit {
     }
 
     function _addFlow(uint256 flowLimit_, uint256 slotToAdd, uint256 slotToCompare, uint256 flowAmount) internal {
-        uint256 flowToAdd;
-        uint256 flowToCompare;
+        uint256 flow;
+        uint256 reverseFlow;
 
```

Comment:
the only difference is epoch time in this test contract. can that be made into an internal function that gets overridden in the test contract when inheriting FlowLimit.sol? removes all this duplication